### DISCUSSION TITLE
We are trying to run dotnet publish on our integration-test project to run artifact on other "machine", but get no output

### DISCUSSION THREAD

**Discussion by ThomasCronholm** [microsoft/aspire#6748] [Q&A]
### We have a scenario where we want to build a integration-test project and then move it to the "machine" where the tests should run 
The issue we are seeing is that if we build and try `dotnet publish integration-test-project-path --no-build -c Release /p:Version=0.2.0-init-repo0021 -v m` we get no publish output.
If we build and try
-> copy the build output to folder
-> zip the folder
-> upload it to build artifacts
-> download and extract it on the test-machine and run our tests
We get `System.IO.DirectoryNotFoundException : E:\home\vsts\work\1\s\Source\OurApplication.AppHost\` when we have `/TestAdapterPath:"E:\AzureAgents\nameofmachine_192.168.0.0.1.1\_work\52\s`.

We have seen that a reference to the aspire AppHost project disallows the publish.
The question is how can we accomplish this?

**Comment by japj** [comment]
Does https://github.com/dotnet/aspire/discussions/1819 answer your question?
There is a link to aspirate and a video explaining things

**Comment by ThomasCronholm** [comment]
Thank you for your answer.
No since I can not get aspirate to work in the most basic of examples.
![image](https://github.com/user-attachments/assets/e96910ba-f98c-43ff-8f0f-3e758ec70418)

We have pivoted to try and run integration tests from a container based on your suggestion.
But we would like to automate as much as possible, and not getting aspirate to work gives us pause.
We are running Aspire 9, .Net 9, and aspirate 8.0.7.
